.github/workflows/test-rapidwright-wrapper.yml

```diff
@@ -13,11 +13,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Setup JDK 1.11
+      - name: Setup JDK 1.17
```

Comment:
Not resistant to this change -- but is there a reason we should document behind moving to 17 (and why not the latest LTS?)

Reply:
I switched this when I was trying to diagnose CI failures as Elk is being distributed with Java 17 jar builds.  The other CI test also uses Java 17 so I thought we should be consistent.  I resolved the issue by posting equivalent Java 8 jars of the Elk dependencies and use those if <Java 17 is detected.